proto/internal/temporal/server/api/matchingservice/v1/request_response.proto

```diff
@@ -305,13 +308,30 @@ message GetTaskQueueUserDataRequest {
     // available (or timeout).
     bool wait_new_data = 4;
 }
+
 message GetTaskQueueUserDataResponse {
     reserved 1;
     // Versioned user data, set if the task queue has user data and the request's last_known_user_data_version is less
     // than the version cached in the root partition.
     temporal.server.api.persistence.v1.VersionedTaskQueueUserData user_data = 2;
 }
 
+message SyncDeploymentUserDataRequest {
```

Comment:
maybe `SyncTaskQueueDeploymentDataRequest`? I think it's appropriate for the name to have TaskQueue in it.

Reply:
Sure, if you like this one better. But every other API in matching service has TaskQueue in it's name :)

Reply:
except `UpdateWorkerBuildIdCompatibility` and `UpdateWorkerVersioningRules`, which are the two most similar to this one :man_shrugging: